.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "restricted",
```

Comment:
For our goals, should we update `access` to `public`?

https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#access-restricted--public